projects/zkera/index.js

```diff
@@ -0,0 +1,9 @@
+const { gmxExports } = require("../helper/gmx");
+
+const vault = "0x17D3FdF3b017C96782dE322A286c03106C75C62E";
```

Comment:
but this is zkdx vault? https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/zkdx-finance/index.js#L17

Reply:
Our users try to add our info to defillama without our devs, so there is a conflict rn
you can check "Tvl Code"on the https://defillama.com/protocol/zkera-finance#information page, in this way you can see that this isn't zkdx project, it's zkEra listed by somebody else

Reply:
Our users try to add our info to defillama without our devs, so there is a conflict rn
you can check "Tvl Code"on the https://defillama.com/protocol/zkera-finance#information page, in this way you can see that this isn't zkdx project, it's zkEra listed by somebody else

Reply:
okay, but if it is already called zkera on the app, why this PR?

Reply:
to fix the folder name?

Reply:
We didn't know about other deployment and want to be in charge of any future updates, what should we do in this case?